### PR TITLE
Fixes #420 add yaydoc.yml

### LINKED ISSUE #420: Add yaydoc.yml file for yaydoc
Yaydoc will need a yaml file named `.yaydoc.yml` with various configuration.

### PR DESCRIPTION
**Pull request by pri22296**
Fixes #420

### CODE REVIEW

**Comment by pri22296** [comment]
@imujjwal96 @sch00lb0y @niranjan94 Please review

**Comment by pri22296** [comment]
Preview: http://yaydoc.herokuapp.com/preview/priyamsingh.22296@gmail.com/960aa29a-2c1f-4820-b602-27d2a52ae7ac_preview/

**Comment by mejariamol** [comment]
@pri22296 
>Preview: http://yaydoc.herokuapp.com/preview/priyamsingh.22296@gmail.com/960aa29a-2c1f-4820-b602-27d2a52ae7ac_preview/

the link is giving 404. please take a look.

**Comment by iamareebjamal** [comment]
@mejariamol It is a temporary link. It will expire after few hours of posting

**Comment by pri22296** [comment]
@mejariamol @iamareebjamal Here's an updated link

https://yaydoc.herokuapp.com/preview/priyamsingh.22296@gmail.com/a90b1d61-df2f-4610-8666-b48a2c9df9a2_preview/

File: .yaydoc.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  version: development
+
+build:
+  source: docs
+  theme:
+    name: sphinx_fossasia_theme
+
+extras:
+  javadoc:
+    path: app/src
```
Comment (sch00lb0y) [inline .yaydoc.yml]: java doc is not necessary I guess

Comment (pri22296) [inline .yaydoc.yml]: @sch00lb0y So should I remove it?

Comment (sch00lb0y) [inline .yaydoc.yml]: since javadoc is not working for the android app. I think there is no necessary of having this PR. so please close this PR

**Comment by pri22296** [comment]
Closing this on suggestion of @sch00lb0y

**Pull request closed without merging**